app/(tabs)/index.tsx

```diff
@@ -1,122 +1,352 @@
+    },
+    {
+      id: 3,
+      title: 'FLASH (2023)',
+      image: require('@/assets/images/movies/15120971aa7848def590eaeeda16dddc64d4fe45.jpg'),
+    },
+    {
+      id: 4,
+      title: 'AQUAMAN',
+      image: require('@/assets/images/movies/762390e133aef26ddd029fc8c7cad1e3bbfccd99.jpg'),
+    },
+    {
+      id: 5,
+      title: 'GUARDIANS',
+      image: require('@/assets/images/movies/900980cc012e8a892443f1ffc4b1045b1e124173.jpg'),
+    },
+  ]
+
+  const renderMovieCard = ({ item }: { item: any }) => (
+    <View style={styles.movieCard}>
+      <Image source={item.image} style={styles.moviePoster} />
+      <View style={styles.playOverlay}>
+        <Ionicons name='play' size={24} color='#FFFFFF' />
+      </View>
+      <ThemedText style={styles.movieTitle}>{item.title}</ThemedText>
+    </View>
+  )
+
   return (
-    <ParallaxScrollView
-      headerBackgroundColor={{ light: '#A1CEDC', dark: '#1D3D47' }}
-      headerImage={
-        <Image
-          source={require('@/assets/images/partial-react-logo.png')}
-          style={styles.reactLogo}
-        />
-      }
-    >
-      <ThemedView style={styles.titleContainer}>
-        <ThemedText type='title'>Welcome!</ThemedText>
-        <HelloWave />
-        <ThemedText type='subtitle' style={styles.userInfo}>
-          Hello{user ? `, ${user.firstName}` : ''}!
-        </ThemedText>
-      </ThemedView>
-      <ThemedView style={styles.stepContainer}>
-        <ThemedText type='subtitle'>Step 1: Try it</ThemedText>
-        <ThemedText>
-          Edit{' '}
-          <ThemedText type='defaultSemiBold'>app/(tabs)/index.tsx</ThemedText>{' '}
-          to see changes. Press{' '}
-          <ThemedText type='defaultSemiBold'>
-            {Platform.select({
-              ios: 'cmd + d',
-              android: 'cmd + m',
-              web: 'F12',
-            })}
-          </ThemedText>{' '}
-          to open developer tools.
-        </ThemedText>
-      </ThemedView>
-      <ThemedView style={styles.stepContainer}>
-        <Link href='/modal'>
-          <Link.Trigger>
-            <ThemedText type='subtitle'>Step 2: Explore</ThemedText>
-          </Link.Trigger>
-          <Link.Preview />
-          <Link.Menu>
-            <Link.MenuAction
-              title='Action'
-              icon='cube'
-              onPress={() => alert('Action pressed')}
-            />
-            <Link.MenuAction
-              title='Share'
-              icon='square.and.arrow.up'
-              onPress={() => alert('Share pressed')}
-            />
-            <Link.Menu title='More' icon='ellipsis'>
-              <Link.MenuAction
-                title='Delete'
-                icon='trash'
-                destructive
-                onPress={() => alert('Delete pressed')}
-              />
-            </Link.Menu>
-          </Link.Menu>
-        </Link>
+    <ThemedView style={styles.container}>
```

Comment:
Here you should use the ThemedSafeAreaView, so it has the correct background depending on the theme, and also fill the screen correctly.
```
<ThemedSafeAreaView style={commonStyles.screenContainer}>
```